src/pyfaf/utils/storage.py

```diff
@@ -19,37 +19,30 @@
 
 # Utility functions related to faf's storage
 
-import re
 from collections import defaultdict
 
 __all__ = ["format_reason", "most_common_crash_function"]
 
-RE_SIGNAL = re.compile("SIG[^)]+")
-
 
 def format_reason(rtype, reason, function_name):
     """
     Return formatted `reason` of the crash according to report type `rtype`
     """
 
-    if rtype == "USERSPACE":
-        res = RE_SIGNAL.search(reason)
-        if res:
-            return "{0} in {1}".format(res.group(), function_name)
-
+    if rtype == "userspace":
```

Comment:
I think this should be `core`.

https://github.com/abrt/faf/blob/c7927635e89189dd70bf840498121f6f7c6ab2f4/src/pyfaf/ureport_compat.py#L35

Reply:
Yup, thanks.